base/daemon/BaseDaemon.cpp

```diff
@@ -85,7 +85,12 @@ const char * msan_strsignal(int sig)
     // Apparently strsignal is not instrumented by MemorySanitizer, so we
     // have to unpoison it to avoid msan reports inside fmt library when we
     // print it.
+#if __GLIBC__ > 2 || (__GLIBC__ == 2  && __GLIBC_MINOR__ >= 32)
```

Comment:
AFAIK we are using frozen version of libc headers (see `contrib/libc-headers` and `cmake/toolchain`) on every platform, so there is no chance that `__GLIBC_MINOR__` will change.

Also glibc has no intention to break ABI anytime soon, so `sys_siglist` will continue to work.

Reply:
>AFAIK we are using frozen version of libc headers (see contrib/libc-headers and cmake/toolchain) on every platform, so there is no chance that __GLIBC_MINOR__ will change.

For bundled build - yes.

>Also glibc has no intention to break ABI anytime soon, so sys_siglist will continue to work.

ABI - yes, but it does not included into string.h anymore.

>Do I understand correctly that everything is related to the "unbundled" (unfavorable) build?

Yes, this is only for unbundled build

Reply:
I've updated the title to highlight that it is related to unbundled build only.